pxl/config.py

```diff
@@ -45,6 +47,7 @@ def from_json(cls, json: Dict[str, Any]) -> Config:
             deploy_host=json["deploy_host"],
             deploy_user=json["deploy_user"],
             deploy_path=json["deploy_path"],
+            public_image_url=json.get("public_image_url", ""),
```

Comment:
Is the empty string the right default?

Reply:
Not sure, but it's the best I could think of without changing the type of the config to something like `Dict[str, Optional[str]]`.

Reply:
The empty string works because it's a falsy value (`(not not "") == True`), which then gets checked [in build.py](https://github.com/svsticky/pxl/pull/50/files#diff-c78dc65856637afad701a0cb80a11748R29).

Reply:
How the fuck is the empty string falsey. What a language

Reply:
¯\_(ツ)_/¯

Reply:
https://docs.python.org/3/library/stdtypes.html?highlight=str#truth-value-testing